source/AI.cpp

```diff
@@ -1918,7 +1918,8 @@ bool AI::MoveTo(Ship &ship, Command &command, const Point &targetPosition,
 	bool isFacing = (dp.Unit().Dot(angle.Unit()) > .95);
 	if(!isClose || (!isFacing && !shouldReverse))
 		command.SetTurn(TurnToward(ship, dp));
-	if(isFacing)
+	if(isFacing && (ship.Velocity().Length() <= ship.MaxVelocity() * 0.99
+					|| (dp.Unit().Dot(ship.Velocity().Unit()) < .95)))
```

Comment:
What is the impact of this?

Reply:
The ship will stop thrusting if it is close to enough to max speed and is flying in the right direction

Reply:
Will do that

Reply:
done